hack/update-bazel.sh

```diff
@@ -114,3 +114,4 @@ fi
 
 # Avoid bazel stuff in tools/ directory
 rm hack/tools/BUILD
+find hack/tools/logcheck/ -name "BUILD" -type f  -delete
```

Comment:
Should we just run this against everything in `hack/tools` and replace the first command? Is there something that needs it?

Reply:
bazel is gone :)